src/main/java/nl/praegus/fitnesse/symbols/MavenProjectVersions/DependencyInfo.java

```diff
@@ -0,0 +1,66 @@
+package nl.praegus.fitnesse.symbols.MavenProjectVersions;
+
+import org.apache.maven.model.Model;
+import org.w3c.dom.Document;
+
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.xpath.XPathExpression;
+import javax.xml.xpath.XPathFactory;
+import java.net.URL;
+import java.util.regex.Pattern;
+
+public class DependencyInfo {
+    private final static String latestversionXpath = "/metadata/versioning/latest";
+    private String artifactid;
+    private String version;
+    private String latest;
```

Comment:
Ik denk dat we hier nog een stukje naamgeving kunnen verbeteren. We hebben het in latest en in version allebei over versienummers. Is het dan niet beter om te spreken van: versionInPom en versionOnMavenCentral? Dat is weer een stuk duidelijker dan version en latest.

Reply:
Oh en artifact id zijn 2 woorden, dus artifactId, met een hoofdletter i :-)

Reply:
Naamgeving aangepast naar de namen die je hebt aangegeven :)